index.js

```diff
@@ -35,7 +35,7 @@ module.exports = user => {
 
 	return getPackages(user)
 		.then(data => Promise.all(data.map(el => getDependents(el).then(res => {
-			if (!res.length) {
+			if (!res.length > 0) {
```

Comment:
This is weird. Do `res.length === 0` instead.

Reply:
Yepp, forgot my glasses...